.sbtopts

```diff
@@ -0,0 +1,7 @@
+-J-Xmx5g
```

Comment:
Interesting, I haven't used this file before.

Reply:
I was having some trouble executing the tests due to a lack of memory in sbt JVM. I have imported it from another project.